homeassistant/components/russound_rnet/media_player.py

```diff
@@ -85,17 +86,22 @@ def __init__(self, hass, russ, sources, zone_id, extra):
         self._attr_name = extra["name"]
         self._russ = russ
         self._attr_source_list = sources
-        self._zone_id = zone_id
+        self._controller_id = math.ceil(zone_id / 6)
+        self._zone_id = zone_id - ((self._controller_id - 1) * 6)
```

Comment:
Can we use the modulus operator, or is that less readable?
```suggestion
        self._zone_id = (zone_id - 1) % 6 + 1
```

Reply:
Modulus is less readable to me but I don't have a strong opinion here.